src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Shapes/Given_Path.cs

```diff
@@ -41,5 +40,122 @@ public void Should_Not_Include_Control_Points_Bounds()
 			Assert.IsTrue(Math.Abs(50 - SUT.DesiredSize.Height) <= 1, $"Actual size: {SUT.DesiredSize}");
 #endif
 		}
+
+		[TestMethod]
+		[UnoWorkItem("https://github.com/unoplatform/uno/issues/18694")]
+#if !__SKIA__
+		[Ignore("PathFigure.IsFilled's interaction with Path is only implemented on Skia.")]
+#endif
+		public async Task When_PathGeometry_Figures_Not_Filled_ColorBrush()
+		{
+			var SUT = new Path
+			{
+				Fill = new SolidColorBrush(Microsoft.UI.Colors.Red),
+				Data = new PathGeometry
+				{
+					Figures = new PathFigureCollection
+					{
+						new PathFigure
+						{
+							StartPoint = new Point(0, 0),
+							IsFilled = true,
```

Comment:
If the property is not respected on non-Skia targets, does it have `[NotImplemented]` attribute for those targets?